README.md

```diff
@@ -41,9 +41,19 @@ These LLMs are all licensed for commercial use (e.g., Apache 2.0, MIT, OpenRAIL-
 
 ## LLM datasets for fine-tuning
 
-PENDING
+| Name | Release Date |  Paper/Blog | Dataset | Size | License |
+| --- | --- | --- | --- | --- | ---- | 
+| MPT-7B-Instruct | 2023-05-05 | [Introducing MPT-7B: A New Standard for Open-Source, Commercially Usable LLMs](https://www.mosaicml.com/blog/mpt-7b) | [dolly_hhrlhf](https://huggingface.co/datasets/mosaicml/dolly_hhrlhf) | 59,310 | CC BY-SA-3.0 |
+| starcoderdata | 2023-05-04 | [StarCoder: A State-of-the-Art LLM for Code](https://huggingface.co/blog/starcoder) | [starcoderdata](https://huggingface.co/datasets/bigcode/starcoderdata) | 882GB | Apache 2.0 |
+| OpenAssistant Conversations Dataset | 2023-04-14 | [OpenAssistant Conversations - Democratizing Large Language Model Alignment](https://drive.google.com/file/d/10iR5hKwFqAKhL3umx8muOWSRm7hs5FqX/view) | [oasst1](https://huggingface.co/datasets/OpenAssistant/oasst1) | 161,443 | Apache 2.0 |
+| databricks-dolly-15k | 2023-04-12 | [Free Dolly: Introducing the World's First Truly Open Instruction-Tuned LLM](https://www.databricks.com/blog/2023/04/12/dolly-first-open-commercially-viable-instruction-tuned-llm) |  [databricks-dolly-15k](https://huggingface.co/datasets/databricks/databricks-dolly-15k) | 15,000 |  CC BY-SA-3.0 |
```

Comment:
Thanks for this but maybe specify the size 15K to samples, MB?

Reply:
@Muhtasham Went ahead and did that. Also updated the date format.

Reply:
@lawwu Thanks!